README.md

```diff
@@ -0,0 +1 @@
+[Tech Spec](https://cpslo.sharepoint.com/:w:/s/csc308group6/ERqXhPTlsMxMkaaFmAbt31UB_TW3lnbgPoEV74getSyC7A?e=3JYtPA) (viewable only from a Cal Poly account)
```

Comment:
Change Tech to Product

Reply:
pushed a commit to fix. please resolve this conversation if the fix is appropriate